packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -40,10 +43,13 @@
             name="address-locality"
             :label-text="$t('labels.locality')"
             :has-error="isAddressLocalityEmpty"
+            aria-describedby="locality-error"
+            @blur="isFieldValid('locality')"
```

Comment:
@oliversweb is working on a similar ticket, to see when we trigger the validation exactly. Can the two of you please check you're aligned on this, to ensure we're not doing things twice or one PR will override what the other did? Thanks!